config/checker_severity_map.json

```diff
@@ -50,9 +51,11 @@
   "bugprone-argument-comment":                                  "LOW",
   "bugprone-assert-side-effect":                                "MEDIUM",
   "bugprone-bool-pointer-implicit-conversion":                  "LOW",
+  "bugprone-bad-signal-to-kill-thread":                         "MEDIUM",
```

Comment:
Please insert new items into alphabetical order. This way we can avoid to have multiple entries for the same checker (see #2719 as an example). Please check the other new entries too.